assets/src/animation/constants.js

```diff
@@ -104,6 +104,26 @@ export const DIRECTION = {
   LEFT_TO_RIGHT: 'leftToRight',
 };
 
+export const SCALE_DIRECTION = {
+  SCALE_IN: 'scaleIn',
+  SCALE_OUT: 'scaleOut',
+  SCALE_OUT_TOP_RIGHT: 'scaleOutTopRight',
+  SCALE_OUT_BOTTOM_LEFT: 'scaleOutBottomLeft',
+  SCALE_IN_TOP_LEFT: 'scaleInTopLeft',
+  SCALE_IN_BOTTOM_RIGHT: 'scaleInBottomRight',
+};
```

Comment:
This is two fold; we are mapping numeric values to a string identifier of `in` and `out`. However visually there are four radio buttons, so there is a second mapping as well.